lib/Graph/NodeValue.cpp

```diff
@@ -51,6 +52,12 @@ void NodeValue::replaceAllUsesOfWith(NodeValue v, const Function *F) const {
     assert(site->getNode() == node_ && "Invalid user");
     assert(site->getResNo() == getResNo() && "Invalid list of uses");
     site->setOperand(v.getNode(), v.getResNo());
+
+    // Log the change of node input(operand).
+    if (getNode()->getParent()) {
```

Comment:
minor thing, but it's a bit strange that the log is written before calling setOperand above in the FunctionConverter but after here in replaceAllUsesWith. Lets pick one and be consistent.